plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/DockerInstance.java

```diff
@@ -104,7 +105,7 @@
     private final MachineRuntimeInfoImpl                      machineRuntime;
 
     @Inject
-    public DockerInstance(DockerConnector docker,
+    public DockerInstance(DockerConnectorProvider dockerProvider,
```

Comment:
@garagatyi  For initial implementation I would rather leave it this way. The usage of `DockerConnectorProvider` will be changed once `OpenShiftConnector` is decoupled from `DockerConnector` and there will be a good time for rethinking the whole approach